corehq/apps/reports/standard/deployments.py

```diff
@@ -319,3 +323,82 @@ def _bootstrap_class(obj, severe, warn):
         return "label label-warning"
     else:
         return "label label-success"
+
+
+class ApplicationErrorReport(GenericTabularReport, ProjectReport):
+    name = ugettext_noop("Application Error Report")
```

Comment:
Yeah, not sure why the one was used over the other in this instance.  `ugettext_lazy` is nice because it's automatically translated whenever it's passed in to a django template, but it's passed around as a promise type object, which can cause problems (you can't print it or dump it to json without forcing the promise).  Conversely, `ugettext_noop` is literally a noop (`"foo" == ugettext_noop("foo")`), so you're passing around the original string.  This can be nice for logging, but it'll need to be explicitly translated whenever you DO want the translated version of the string.